packages/browserslist-config/package.json.md

```diff
@@ -0,0 +1,33 @@
+# Notes on Packages
+
+- [Version](#version)
+- [Scripts](#scripts)
+
+## Version
+
+Major version number should be following major version of `browserslist`
+in `frontend-config/package.json`.
+
+## Scripts
+
+### `browserslist:*`
```

Comment:
@101100 I did _not_ want to add this comment to `README.md` on purpose, because readme is distributed as part of the NPM package, and this comment does not make sense in the final version of the package (published `package.json` does not have either `scripts` property or `browserslist` property, as both only make sense at development time in `frontend-config` repository).

I did update the browserslist specification/query in the `README.md` file, thank you, good catch.

Reply:
Ah, I guess it is developer convenience vs. nicer documentation on NPM. This works for me.